host-base.json

```diff
@@ -5,7 +5,8 @@
     "repos": [
         "ostree-master",
         "origin-repo",
-        "dustymabe-ignition"
+        "dustymabe-ignition",
```

Comment:
Do we want these in the base or in `host-rhcos.json`?

Reply:
I think these are all RPMs that would be shared between the two variants, so it makes more sense in the base?